src/timely-util/src/containers.rs

```diff
@@ -474,3 +476,75 @@ mod provided_builder {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use mz_ore::region::Region;
+    use timely::bytes::arc::BytesMut;
+    use timely::dataflow::channels::ContainerBytes;
+    use timely::Container;
+
+    use super::*;
+
+    /// Produce some bytes that are in columnar format.
+    fn raw_columnar_bytes() -> Vec<u8> {
+        let mut raw = Vec::new();
+        raw.extend(12_u64.to_le_bytes()); // length
+        raw.extend(1_i32.to_le_bytes());
+        raw.extend(2_i32.to_le_bytes());
+        raw.extend(3_i32.to_le_bytes());
+        raw.extend([0, 0, 0, 0]); // padding
+        raw
```

Comment:
I tried using `Sequence::encode` here, but it gives me a `Vec<u64>` and I don't know how to convert that into a `Vec<u8>` without unsafe, so I just went the manual way.

Reply:
You could use bytemuck: https://docs.rs/bytemuck/latest/bytemuck/fn.cast_slice.html